netmaster/objApi/objapi_test.go

```diff
@@ -2245,6 +2260,29 @@ func get(getAll bool, hook func(id string) ([]core.State, error)) func(http.Resp
 	}
 }
 
+func createEPinEPG(t *testing.T, providerIP, network, epg, containerID, tenant, endpointID string, labels []string) {
+
+	epCfg := &mastercfg.CfgEndpointState{
+		NetID:            network,
+		EndpointID:       endpointID,
+		IPAddress:        providerIP,
+		EndpointGroupKey: epg + ":" + tenant,
+	}
+	epCfg.Labels = make(map[string]string)
+	for _, v := range labels {
+		key := strings.Split(v, "=")[0]
+		value := strings.Split(v, "=")[1]
+		epCfg.Labels[key] = value
+	}
+	epCfg.StateDriver = stateStore
+	netID := network + "." + tenant
+	epCfg.ID = netID + "-" + endpointID
+	err := epCfg.Write()
+	if err != nil {
+		t.Errorf("Error creating Ep :%s", err)
+	}
+}
```

Comment:
Do we need to read the data to ensure if it is really written?

Reply:
not here, it's already an existing field